src/models/app_data_json_format.rs

```diff
@@ -24,13 +26,11 @@ pub struct QuoteV1 {
     pub sell_amount: U256,
     #[serde(with = "u256_decimal")]
     pub buy_amount: U256,
-    pub version: String,
 }
 
 #[derive(Eq, PartialEq, Clone, Debug, Deserialize, Serialize, Hash)]
 #[serde(rename_all = "camelCase")]
```

Comment:
From what I understand, this should be:
```suggestion
#[serde(deny_unknown_fields, rename_all = "camelCase")]
```